### ISSUE TITLE
Cannot Import files produced by UAModeler from UAExpert

### ISSUE THREAD

**Issue by jwise-mfg** [cesmii/ProfileDesigner#40]
The attached files were produced using UAModeler from UAExpert. Importing them in Profile Designer results in an Import Error as shown in attached screen shot
[TestNodesets.zip](https://github.com/cesmii/ProfileDesigner/files/9395854/TestNodesets.zip)
![ImportError](https://user-images.githubusercontent.com/57234292/185966159-6799df85-13c3-4bc5-b383-31fc69906610.png)
.

**Comment by MarkusHorstmann** [comment]
Import of either profile works for me locally, and in both stage and production. Are you still seeing this?

**Comment by jwise-cesmii** [comment]
Yes, screenshot taken this morning. perhaps its a problem with the user's account? (Shown in screen shot)

**Comment by MarkusHorstmann** [comment]
Yes, the user account was created without any permissions. Please try again now.
